src/Bundle/test/src/Entity/ScienceBook.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Sylius\Component\Resource\Model\ResourceInterface;
+
+class ScienceBook implements ResourceInterface
+{
+    private ?int $id;
+
+    private ?Author $author = null;
+
+    private ?string $title = null;
+
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    public function getTitle(): ?string
+    {
+        return $this->title;
+    }
+
+    public function setTitle($title): void
+    {
+        $this->title = $title;
+    }
+
+    public function getAuthorFirstName(): ?string
+    {
+        return $this->author ? $this->author->getFirstName() : null;
```

Comment:
```suggestion
        return $this->author?->getFirstName();
```

Reply:
I have been waiting for this syntax for so long...
It's annoying especially when using Doctrine and everything is nullable.